### PR TITLE
Unique log policy name

### PR DESCRIPTION
**Pull request by JoelClemence**
- Sets log group policy name to unique name
- Corrects the trigger for Lambda
- Improves logging for S3 copier lambda
- Handle the URL encoded object keys

### CODE REVIEW

File: lambdas/ingest-upload-file-lambda/src/main/java/uk/gov/ukho/ais/ingestuploadfile/s3/S3ObjectCopier.java [hunk 1]
```diff
@@ -25,12 +25,13 @@ public S3ObjectCopier(
 
   public boolean copyS3ObjectToNewLocation(final S3Object s3Object) {
     try {
+      LOGGER.info("Copying {} to {}", s3Object.toString(), destinationBucket);
```
Comment (eganjs) [inline lambdas/ingest-upload-file-lambda/src/main/java/uk/gov/ukho/ais/ingestuploadfile/s3/S3ObjectCopier.java]: Shouldn't need an explicit `toString()`
```suggestion
      LOGGER.info("Copying {} to {}", s3Object, destinationBucket);
```

File: lambdas/ingest-upload-file-lambda/src/main/java/uk/gov/ukho/ais/ingestuploadfile/s3/S3ObjectCopier.java [hunk 2]
```diff
@@ -25,12 +25,13 @@ public S3ObjectCopier(
...
       final String destinationKey = destinationKeyPrefix + getFileNameFromKey(s3Object.getKey());
       amazonS3Client.copyObject(
           s3Object.getBucket(), s3Object.getKey(), destinationBucket, destinationKey);
       return true;
     } catch (AmazonServiceException e) {
-      LOGGER.error("Unable to copy file", e);
+      LOGGER.error("Unable to copy file '" + s3Object.toString() + "' to " + destinationBucket, e);
```
Comment (eganjs) [inline lambdas/ingest-upload-file-lambda/src/main/java/uk/gov/ukho/ais/ingestuploadfile/s3/S3ObjectCopier.java]: Shouldn't need an explicit `toString()`

```suggestion
      LOGGER.error("Unable to copy file '" + s3Object + "' to " + destinationBucket, e);
```

File: lambdas/ingest-upload-file-lambda/src/main/java/uk/gov/ukho/ais/ingestuploadfile/s3/S3ObjectCopier.java (hunk 1 above)
Comment (JoelClemence) [reply lambdas/ingest-upload-file-lambda/src/main/java/uk/gov/ukho/ais/ingestuploadfile/s3/S3ObjectCopier.java]: done

File: lambdas/ingest-upload-file-lambda/src/main/java/uk/gov/ukho/ais/ingestuploadfile/s3/S3ObjectCopier.java (hunk 2 above)
Comment (JoelClemence) [reply lambdas/ingest-upload-file-lambda/src/main/java/uk/gov/ukho/ais/ingestuploadfile/s3/S3ObjectCopier.java]: done

**Pull request merged**